viite-UI/src/view/RoadAddressProjectForm.js

```diff
@@ -157,16 +157,20 @@
       };
 
       eventbus.on('roadAddress:newProject', function() {
+        currentProject=undefined; //clears old data
+        $("#roadAddressProject").html("");
         rootElement.html(newProjectTemplate());
         jQuery('.modal-overlay').remove();
         addDatePicker();
         applicationModel.setOpenProject(true);
+        projectCollection.clearRoadAddressProjects();
       });
 
       eventbus.on('roadAddress:openProject', function(result) {
         currentProject = result.projects;
+        projectCollection.clearRoadAddressProjects();
         var text = '';
-        _.each(result.projectLinks, function(line){
+        _.each(result.projectLinks, function(line){  //TODO later list of already saved roadlinks has to be saved in  roadaddressprojectcollection.currentRoadSegmentList for reserve button to function properly now saved links are cleared when newones are reserved
```

Comment:
It wasn't included in this task. PO said there will be task  to manage old projects later, so it can be either removed or left there as reminder. To my understanding it will be implemented after we get submitting projects to TR working.